adresy/draft/příklady/2.xml

```diff
@@ -2,12 +2,11 @@
 <adresa
     xmlns="https://ofn.gov.cz/adresy/"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="https://ofn.gov.cz/adresy/ 
-                        https://ofn.gov.cz/adresy/draft/schémata/adresa.xsd"
-    xmlns:x="https://příklad.cz">

```

Comment:
Tady ten namespace se nemůže vymazat, protože je použit v x:poznámka. Pak je soubor nevalidní.

Reply:
to musela být nějaká chybka, ten bych nemazal, opravím

Reply:
á, už vím, validoval jsem si to v saxxonu a upravoval tak aby neřval na neexistující schéma